src/uct/tcp/tcp_iface.c

```diff
@@ -628,9 +708,9 @@ ucs_status_t uct_tcp_query_devices(uct_md_h md,
 
     dir = opendir(netdev_dir);
     if (dir == NULL) {
-        ucs_error("opendir(%s) failed: %m", netdev_dir);
-        status = UCS_ERR_IO_ERROR;
-        goto out;
+        /* When /sys is unavailable, as can be the case in a container,
+         * resort to a good old 'ioctl'.  */
```

Comment:
there is an excessive space at the end of the comment
remove it pls: `.  */` -> `. */`